lib/pacemaker/pcmk_sched_group.c

```diff
@@ -236,7 +232,10 @@ group_internal_constraints(pe_resource_t *rsc)
                                          RSC_DEMOTED, stopped);
 
             pcmk__order_resource_actions(child_rsc, RSC_PROMOTE, rsc,
-                                         RSC_PROMOTED, started);
+                                         RSC_PROMOTED,
```

Comment:
I like using literal flags when possible, to avoid having to back up and see how a variable was set and possibly modified. down/post_down_flags were necessary because they can be conditionally changed.

Reply:
`start` can be conditionally changed (setting the `pe_order_implies_first_printed` much like we set the `pe_order_optional` bit for `down_flags`) but `started` can't

Ack understood, looks good